description.md

```diff
@@ -1,37 +1,44 @@
-AppMap enhances the developer experience by analyzing runtime code behavior before you commit changes and go to production.
+AppMap is a runtime analysis and visualization platform. It analyzes your application's runtime code behavior before your changes go to production, and allows you to understand complex code interactions.
 
-AppMap records detailed traces of how your application runs and analyzes it based on frameworks and techniques including web services, view templates, HTTP client requests, caching, authentication, and SQL.
+## Runtime behavior visualization
+After running your application with the AppMap library, the runtime recordings are instantly visualized in the IDE. These visualizations include:
+* **Sequence Diagrams** to follow the runtime flow of calls made by your application
+* **Dependency Maps** to see which libraries and frameworks were used at runtime
+* **Flame Graphs** to spot performance issues and bottlenecks
+* **Trace Views** to perform detailed function call and data flow tracing
 
-Bugs, loops, calls, dependencies, security issues, and more are found with ease in trace views and sequence diagrams.
+## Runtime flaw analysis
+After making recordings of how your application behaved at runtime, AppMap analyzes those recordings to automatically detect performance issues like N+1 queries, and security flaws such as faulty authentication logic.
+
+## AppMap in CI
+The same features available in this plugin are [also available for CI systems](https://appmap.io/docs/analysis/in-ci.html). AppMap for CI can also analyze each Pull Request and check for flaws, highlight unexpected API changes, provide insights into CI test failures, and display behavioral comparisons of runtime changes.
+
+## How does AppMap work?
```

Comment:
I would drop this whole section. We try and describe this in so many places they all get out of date.

Reply:
It's more for making people aware of the other product, which could be very valuable to someone who only heard of us via the marketplace. Maybe I just remove the last sentence which describes the feature set?